---
string  state
string impedance_ref_link

